docs/reference/cluster_manifest.md

```diff
@@ -33,7 +33,15 @@ Those parameters are grouped under the `metadata` top-level key.
   services, secrets) for the cluster. Changing it after the cluster creation
   results in deploying or updating a completely separate cluster in the target
   namespace. Optional (if present, should match the namespace where the
-  manifest is applied). 
+  manifest is applied).
+
+* **labels**
+  if labels are matching one of the `inherited_labels` [configured in the
+  operator parameters](docs/reference/operator_parameters.md#kubernetes-resources),
```

Comment:
link does not work for me. Just leave out `docs/reference/`.

Reply:
Good catch!

I've pushed a fix. Thanks.